lib/ProductOpener/Producers.pm

```diff
@@ -538,7 +539,7 @@ my %fields_synonyms = (
 
 en => {
```

Comment:
should'nt we migrate that to a taxonomy at some point ? ;-)

Reply:
at some point, yes :)